packages/article-feed-persoo/src/Resources/config/services.yaml

```diff
@@ -0,0 +1,12 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Shopsys\ArticleFeed\PersooBundle\:
+        resource: '../../*'
+
+    Shopsys\ArticleFeed\PersooBundle\PersooArticleFeed:
+        tags:
+            - { name: shopsys.feed, hours: '4', minutes: '0' }
```

Comment:
can be the timing altered from the project?

Reply:
yeah, Tom prepared `feed.yaml` file for that purpose, see https://github.com/shopsys/shopsys/blob/14.0/docs/model/product-feeds.md#when-are-they-exported

> Feeds have their default times set in their own services.yaml files, but can be easily changed in your project's feed.yaml file. You only need to copy service definition from feed services.yaml file and change the hours and minutes to expected ones.